projects.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
Why this line?

Reply:
Just my yaml linter telling me to start a yaml document with it. But we can safely remove it, if you want?

Reply:
It's optional, so I think remove it from the commit to minimize changes.

Reply:
ok, I have force pushed the change.

Reply:
do you want me to change anything else?